### ISSUE TITLE
Error: ReferenceError: name is not defined after 0.0.72 release

### ISSUE THREAD

**Issue by argshook** [applitools/Eyes.Sdk.JavaScript#57]
Hello!

Our builds started to fail after `0.0.72` was released

```
Error: ReferenceError: name is not defined
    at EyesBase.setBatch (/home/builduser/agent00/work/df7e5aad8673267c/node_modules/eyes.sdk/src/EyesBase.js:467:23)
    at _init (/home/builduser/agent00/work/df7e5aad8673267c/node_modules/eyes.it/index.js:124:8)
    at Object.<anonymous> (/home/builduser/agent00/work/df7e5aad8673267c/node_modules/eyes.it/index.js:127:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
```

please let me know if more details are needed

thank you!

**Comment by argshook** [comment]
im guessing blindly, but from code it looks like issue is simple typo, here's PR https://github.com/applitools/Eyes.Sdk.JavaScript/pull/58

**Comment by argshook** [comment]
@danielputerman fyi, sorry for spam :)

**Comment by danielputerman** [comment]
Lol, no spam at all :) Fix available now in version 0.0.73 (eyes.selenium 0.0.83 works with this version).

**Comment by argshook** [comment]
Great, thank you!
